core-api/src/main/java/com/optimizely/ab/internal/ExperimentUtils.java

```diff
@@ -93,26 +99,28 @@ public static Boolean evaluateAudience(@Nonnull ProjectConfig projectConfig,
 
         OrCondition implicitOr = new OrCondition(conditions);
 
-        logger.debug("Evaluating audiences for experiment \"{}\": \"{}\"", experiment.getKey(), conditions);
+        logger.debug("Evaluating audiences for {} \"{}\": {}.", audienceFor, loggingKey, conditions);
```

Comment:
It should be in doesUser...

Reply:
we should not move it to doesUserMeetCondition method because list of conditions object is created here. To move this log we would have to move the loop too.